Makefile

```diff
@@ -29,7 +29,7 @@ LIBDIR_NAME=lib
 SHAREDLIB_DIR=$(PREFIX)/lib
 PROJECT_NAME=openh264
 MODULE_NAME=gmpopenh264
-GMP_API_BRANCH=Firefox114_2
+GMP_API_BRANCH=Firefox135
```

Comment:
I don't see Firefox135 branch at https://github.com/mozilla/gmp-api/branches/all

Reply:
There is a tag which works https://github.com/mozilla/gmp-api/releases/tag/Firefox135

Reply:
> There is a tag which works https://github.com/mozilla/gmp-api/releases/tag/Firefox135

oh, I was tricked by the word BRANCH :)